### DISCUSSION TITLE
https://notes.alexkehayias.com/note-blogging/

### DISCUSSION THREAD

**Discussion by giscus** [alexkehayias/note-comments#5] [Announcements]
# https://notes.alexkehayias.com/note-blogging/

A blog of working notes that others can read and follow along with to learn about interesting ideas and things you are coming across.

https://notes.alexkehayias.com/note-blogging/

**Comment by Kif11** [comment]
I enjoy your blog! Can you provide more details on how did you set up this web site from your org files?

**Reply by alexkehayias** [reply]
Thank you @Kif11! You can find more details about how [exporting org-roam to markdown here](https://github.com/alexkehayias/emacs.d/blob/master/init-export.el#L541) and in the [blog post here](https://www.alexkehayias.com/essays/zettelkasten-setup/). The rest is hugo, so you can grab the css right off of this page if you want.

**Comment by nmsalgueiro** [comment]
Great blog. I'd love to follow it with RSS but can't seem to figure out the address (tried with suffix '/rss' but didn't work). Could you let me know, please?

**Reply by alexkehayias** [reply]
Thanks for the kind words! The RSS feed is on https://notes.alexkehayias.com/index.xml. I'm not sure I configured it properly to get picked up by feed readers if you navigate to the homepage, but hopefully the direct link works for you.

**Reply by nmsalgueiro** [reply]
Sorry for not answering earlier. It works great with Elfeed. Much appreciated!
